03 Writing Algorithms/03 Securities/99 Asset Classes/07 Futures/04 Market Hours/11 CBOT/01 Introduction.html

```diff
@@ -1 +0,0 @@
-<p>This page shows the trading hours, holidays, time zone, and supported assets of the CBOT Future market.</p>
```

Comment:
This PR is missing some of the asset classes. For example, [this US Equity page](https://github.com/QuantConnect/Documentation/blob/master/03%20Writing%20Algorithms/03%20Securities/99%20Asset%20Classes/01%20US%20Equity/09%20Market%20Hours/01%20Introduction.html) should be importing [this resource](https://github.com/QuantConnect/Documentation/blob/master/Resources/datasets/market-hours/equity/usa/generic/introduction.html). Recommend checking all of the asset classes